SingularityUI/app/views/tail.coffee

```diff
@@ -176,7 +178,8 @@ class TailView extends View
         @$el.addClass 'tailing'
 
     stopTailing: ->
-        return if @isTailing isnt true
+        task = @model.get('taskUpdates')[..].pop()
```

Comment:
seems wasteful to copy the array just to pop -- `_.last()` would get the job done too